DigitalLearningSolutions.Data.Tests/Services/TutorialServiceTests.cs

```diff
@@ -64,5 +65,26 @@ public void UpdateTutorialsStatuses_calls_data_services_correct_number_of_times(
                     .MustHaveHappened(2, Times.Exactly);
             }
         }
+
+        [Test]
+        public void GetTutorialsForSection_calls_data_service()

```

Comment:
Could this test name be tweaked so that it describes the whole test scope i.e. data service called and the expected response.

Reply:
yep done :)